com.corifeus.onenote.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=P3X Onenote
+Icon=com.corifeus.onenote
+Exec=/app/bin/run.sh %u
+Comment=https://www.patrikx3.com
```

Comment:
```suggestion
```

Reply:
Upstream uses this too https://github.com/patrikx3/onenote/blob/4dcdc4e2ad59d7454319e5166f144ff185cf4fc9/bin/p3x-onenote.js#L27

Reply:
Any reason for not using the upstream one then?

Reply:
Upstream is generating the desktop file at runtime, as you can see from the link